fem/coefficient.hpp

```diff
@@ -68,6 +70,18 @@ class Coefficient
       return Eval(T, ip);
    }
 
+   /** @brief Evaluate the coefficient at all the quadrature points given by the
```

Comment:
For clarity, the documentation should also say "on all elements".

Reply:
done.